examples/nidaqmx/analog-input-betterproto.py

```diff
@@ -98,6 +99,13 @@ async def raise_if_error(response):
 
         print(f"Acquired {response.samps_per_chan_read} samples.")
         print(f"First 5 samples: {response.read_array[:5]}")
+    except GRPCError as e:
```

Comment:
There's a different implementation of this for the other [betterproto ](https://github.com/ni/grpc-device/blob/0771571470857a699b2562fafa0479fdabe31ee0/examples/niscope/graph-measurement-betterproto.py#L200) examples.

But that doesn't work for me at all (either when I copied the code or running those examples directly). It requires an exception with at least 2 args (the `OSError` and `TimeoutError` that I got for connection failures only had 1) and it looks for a code `22` (where does that come from? I didn't see it). 

@ni-pkhare do you know what's going on with that? Did that logic work for you?

If those are really broken, I can apply the same fix to all of them.

We could also consider an alternative of always just printing `str(e)`. That's the best for readability of the example. But the `UNIMPLEMENTED` error specifically is harder to make sense of (though *very* standard as a grpc error).

Reply:
@astarche I am not sure why it did not work for you, because I tried it today as well and it seems to be working for me.
1. err code 22 is for the case when we run the example without running the server. When I debugged for this case, I saw err code 22, thats why I used it so that we could display a proper message.
![image](https://user-images.githubusercontent.com/78592845/127642339-9dc0d63e-db74-4d22-861b-46a0ef7a42b0.png)

2. I am also a fan of simply printing the error message using str(e). Also the way you have written for Unimplemented case and other cases in general is quite nice. I would go for it.

Reply:
Interesting!

Thanks for the screenshot. I don't have a complete explanation for that. But, I think that's just another example to go with my 3 slightly different failure modes in the PR description. Error reporting for os-level connection failures is just harder to filter on than the grpc-errors that the standard protoc generator gives you. My implementation is a decent way to deal with that for an example. So, I'll got with this proposal and I created #282 to fix this up.